cidc_schemas/schemas/templates/analyses/wes_analysis_template.json

```
@@ -355,6 +341,26 @@
                                     "is_artifact": 1
                                 }
                             ]
+                        },
+                        "Neoantigen MHC Class II": {
+                            "merge_pointer": "/neoantigen_class_ii_analyzed",
+                            "type_ref": "assays/components/ngs/wes/wes_pair_analysis.json#properties/neoantigen_class_ii_analyzed",
+                            "process_as": [
+                                {
+                                    "parse_through": "lambda run: f'analysis/neoantigen/{run}/MHC_Class_II{run}.all_epitopes.tsv'",
```

I think there's a typo
```suggestion
                                    "parse_through": "lambda run: f'analysis/neoantigen/{run}/MHC_Class_II/{run}.all_epitopes.tsv'",
```

---

Also I think this `lambda` here won't get `run` as input but rather this boolean `class_ii_analysed`